source/slang/slang-capabilities.capdef

```diff
@@ -817,6 +817,12 @@ alias atomic_glsl_hlsl_nvapi_cuda9_int64 = atomic_glsl_int64 | hlsl_nvapi + _sm_
 alias atomic_glsl_hlsl_cuda = atomic_glsl | _sm_5_0 | _cuda_sm_2_0;
 alias atomic_glsl_hlsl_cuda9_int64 = atomic_glsl_int64 | _sm_6_6 | _cuda_sm_9_0;
 
+alias helper_lane = _sm_6_0 + fragment
```

Comment:
I checked out the capabilities, it looks like the glsl and spirv capability conjunction can be simplified into `GL_EXT_demote_to_helper_invocation` (since `GL_EXT` defines `_GL_EXT + _GLSL_140` and the spirv is similarly defined)
```
alias helper_lane = _sm_6_0 + fragment
                   | GL_EXT_demote_to_helper_invocation + fragment
                   | metal + fragment
                   ;
```